.github/workflows/tests.yml

```diff
@@ -70,9 +98,14 @@ jobs:
     steps:
     - name: Check out code
       uses: actions/checkout@v3
-
+        
+    - name: Checkout code
+      if: ${{ inputs.repository }} != ${{ github.repository }}
+      uses: actions/checkout@v3
+      with: 
+          repository: UBCSailbot/sailbot_workspace
     - name: Run linter
       uses: gaurav-nelson/github-action-markdown-link-check@v1
       with:
         config-file: .markdown-link-check.json
-        folder-path: .
+        folder-path: .
```

Comment:
Should have empty line at the end of every file

Reply:
Added empty line to end of tests.yml